op-chain-ops/genesis/genesis.go

```diff
@@ -93,8 +98,8 @@ func NewL2Genesis(config *DeployConfig, l1StartHeader *types.Header) (*core.Gene
 
 	extraData := config.L2GenesisBlockExtraData
```

Comment:
Should we remove the ability to configure the extradata now? Given that its now a consensus parameter, feels like a footgun. We could also delete the definition of `BedrockTransitionBlockExtraData`

Reply:
yeah, let me do that.

Reply:
done.